tests/onebox.go

```diff
@@ -516,8 +522,14 @@ func (c *temporalImpl) startHistory(
 			c.logger.Fatal("Failed to create connection for history", tag.Error(err))
 		}
 
+		matchingConnection, err := rpc.Dial(c.MatchingGRPCServiceAddress(), nil, c.logger)
+		if err != nil {
+			c.logger.Fatal("Failed to create connection for history", tag.Error(err))
+		}
```

Comment:
you shouldn't need to do this here, I don't think. you can get it from the clientBean. or ask fx to populate it? but even if you do need this somewhere, this is a weird place for it (in the loop that constructs history services?) maybe in startMatching instead?

Reply:
Hmm.. I'll move it.